index.html

```diff
@@ -1,48 +1,297 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-</head>
+  </head>
 
-<body>
+ <body>
 
-<header>
-    <nav class="menu">
-    <ul class="container1">
-        <li class="logo"><a href="#">Welcome</a></li>
-    <li><i class="fa fa-bars fa-2x"></i></li>
-    </ul>
-    </nav>
+    <header>
+        <nav class="menu">
+        <ul class="container1">
+            <li class="logo"><a href="#">Welcome</a></li>
+        <li><i class="fa fa-bars fa-2x"></i></li>
+        </ul>
+        </nav>
 
-</header>
-<main class="background">
-<section class="container2">
+    </header>
+    <main class="background">
+        <section id="homepage" class="container2">
 
-    <h1 class="greeting">Hey there. I am Houssam <br> <span> I am a software developer </span> </h1>
-    <p class="presentation"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-don’t hestiate to contact me.</p>
+            <h1 class="greeting">Hey there. I am Houssam <br> <span> I am a software developer </span> </h1>
+            <p class="presentation"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
 
-<ul class="connect" >
-    <li class="media"><a href="#"><img alt="twitter" src="twitter.png"></a></li>
-    <li class="media"><a href="#"><img alt="linkedin" src="linkedin.png"></a></li>
-    <li class="media"><a href="#"><img alt="medium" src="medium.png"></a></li>
-    <li class="media"><a href="#"><img alt="github" src="github.png"></a></li>
-    <li class="media"><a href="#"><img alt="angellist" src="angellist.png"></a></li>
-</ul>
+            <ul class="connect" >
+                <li id="twitter" class="media"><a href="#"><img alt="twitter" src="img/twitter.png"></a></li>
+                <li class="media"><a href="#"><img alt="linkedin" src="img/linkedin.png"></a></li>
+                <li class="media"><a href="#"><img alt="medium" src="img/medium.png"></a></li>
+                <li class="media"><a href="#"><img alt="github" src="img/github.png"></a></li>
+                <li class="media"><a href="#"><img alt="angellist" src="img/angellist.png"></a></li>
+            </ul>
 
 
- </section>
- </main>
+        </section>
+
+    
+    
+    </main>
+
  
     
-</body>
+        <header>
+
+            <div><button  class="scroll-down" type="button">
+                <i class="fa fa-angle-down fa-2x"></i>
+            </button></div>
+
+            <div class="recent-works"><h1>My Recent Works</h1></div>
+            <div class="indicator">
+                <img alt="indcator" src="img/Indicator.png">
+            </div>       
+
+        </header>
```

Comment:
- [ ] We don't need `header` tag here, let us wrap the container of the section to be a `section tag`

Reply:
- [ ] **REQUIRED:** As the previous reviewer mentioned, it is not good to use the `header` here, just replace it with the `section` tag.